components/CartSidebar.vue

```diff
@@ -0,0 +1,150 @@
+<template lang="pug">
+  div.cart-sidebar-wrapper(:class="openSidebarCart ? 'active' : ''")
+    div.overlay-cart-sidebar(@click="closeSidebarCart()")
+    aside.cart-sidebar
+        div.cart-sidebar-header
+            h4.title.is-3 Cart
+            i.fa.fa-times.fa-lg(src="/cross.svg" @click="closeSidebarCart()")
```

Comment:
Looks good! Probably src can be removed now:
```suggestion
            i.fa.fa-times.fa-lg(@click="closeSidebarCart()")
```

Reply:
Not actually resolved yet (: